src/main/resources/au/com/centrumsystems/hudson/plugin/buildpipeline/extension/PipelineHeaderExtension/columnHeader.jelly

```diff
@@ -27,11 +27,9 @@
                                             <td id="project-${project.getId()}"></td>
                                         </j:otherwise>
                                     </j:choose>
-                                    <script>
-                                        //save project proxies for future reference (being able to refresh project cards)
-                                        buildPipeline.projectProxies[${project.getId()}] = <st:bind value="${project}" />;
-                                        jQuery("#project-${project.getId()}").append(buildPipeline.projectCardTemplate(${project.asJSON()}));
-                                    </script>
+                                    <st:adjunct includes="au.com.centrumsystems.hudson.plugin.buildpipeline.extension.PipelineHeaderExtension.pipeline-column-headers"/>
+                                    <st:bind value="${project}" var="projectProxy${project.getId()}"/>
```

Comment:
What symbols can `project.getId()` produce? Are we sure it will always give us symbols that can be used in variable names?

Reply:
I'm not sure I understand exactly what you mean. The java method returns an Integer. 

Can you explain in more detail how this might bring up some issues?

Reply:
I mean if `getId` returned something with dashes that won't be a valid JS variable name. Just integers are fine.